.github/workflows/nodejs.yml

```diff
@@ -9,6 +9,9 @@ on:
     branches:
       - '**'
 
+env:
+  CI: true
```

Comment:
@hudochenkov I think I already mentioned in PR when we was adding github actions, I did not find who is reading this env.